### PR TITLE
Update SPS multiscale example with one-way coupling from engineering scale to phase field

### LINKED ISSUE #83: Update SPS example input files for one-way coupling from engineering scale to phase-field
## Reason
The current example SPS one-way coupling example (where engineering-scale temperatures are passed to a phase-field model) causes temperatures in the to rise outside the calibration range of thermal conductivity for stainless steel. Also, the input file for the phase-field model needs many updates to be consistent with latest model developments.

## Design
Changing initial temperatures and boundary conditions in the engineering-scale simulation to be more realistic as suggested by @sapitts. Changes to syntax and some parameters for phase-field model.

## Impact
Only tested with RunApp, so no changes beyond input files themselves.

### PR DESCRIPTION
**Pull request by laagesen**
Closes #83

### CODE REVIEW

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i [hunk 1]
```diff
@@ -36,7 +36,7 @@
 # Reference for graphite, stainless steel: Cincotti et al, DOI 10.1002/aic.11102
 # Assorted references for yttria, listed as comments in input file
 
-initial_temperature=873 #roughly 600C where the pyrometer kicks in
+initial_temperature = 873 #roughly 600C where the pyrometer kicks in
```
Comment (cticenhour) [inline examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i:39]: If we're no longer using this variable in the input, do we want to keep it around at all?

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i [hunk 2]
```diff
@@ -56,7 +56,7 @@ initial_temperature=873 #roughly 600C where the pyrometer kicks in
 
 [Variables]
   [temperature_stainless_steel]
-    initial_condition = ${initial_temperature}
+    initial_condition = 300
```
Comment (cticenhour) [inline examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: Please add a comment on units.

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i [hunk 3]
```diff
@@ -452,7 +451,7 @@ initial_temperature=873 #roughly 600C where the pyrometer kicks in
     type = CoupledConvectiveHeatFluxBC
     boundary = water_channel
     variable = temperature_stainless_steel
-    T_infinity = ${initial_temperature}
+    T_infinity = 300
```
Comment (cticenhour) [inline examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: Please comment on units

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i [hunk 4]
```diff
@@ -107,7 +107,6 @@ initial_temperature=873 #roughly 600C where the pyrometer kicks in
     block = 'powder_compact'
   []
 
-
   # [T_infinity]
   #   initial_condition = ${initial_temperature}
   # []
```
Comment (cticenhour) [inline examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: Should this also be removed?

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Comment by laagesen** [comment]
thanks for the review @cticenhour, I will address shortly!

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i (hunk 1 above)
Comment (laagesen) [reply examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i:39]: this variable still gets used as IC for blocks other than the stainless steel

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i (hunk 2 above)
Comment (laagesen) [reply examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: done

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i (hunk 3 above)
Comment (laagesen) [reply examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: added comment that units are K

File: examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i (hunk 4 above)
Comment (laagesen) [reply examples/sps/multiapp/electrothermal_with_phase_field/oneway_controls/engineering_scale_electrothermal_oneway_controls.i]: done

**Review by cticenhour — APPROVED** [review]
Thanks for the update @laagesen!

**Pull request merged**